docs/src/content/docs/integrations/ucs01/cosmwasm/index.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: "UCS01 Relay Rust Integration"
+description: "Example showcasing how to create a CosmWasm contract that leverages UCS01 Relay to execute cross-chain asset transfer."
+---
+
+import Code from '#/components/Code.astro'
+import { Tabs, TabItem, FileTree, Steps } from '@astrojs/starlight/components'
+
+This example demonstrates how to create a CosmWasm contract that calls Union's UCS01 Relay to execute a cross-chain asset transfer.
+
+```sh frame="none"
+UCS01_RELAY=
+```
+
+Source code for this example can be found her: [example-ucs01-cosmwasm](https://github.com/unionlabs/example-ucs01-cosmwasm)
+
+For this example we will be using [Rust](https://rustup.rs) to write the smart contract and [CosmWasm](https://cosmwasm.com) as the smart contract platform.
```

Comment:
```suggestion
For this example we will be using [Rust](https://rustup.rs) and [CosmWasm](https://cosmwasm.com).
```

Reply:
These docs already target devs who either know what Rust and CosmWasm and continue to read, or don't and stop reading. So no need to increase the read time IMO.